vassal-app/src/main/java/VASSAL/configure/BooleanConfigurer.java

```diff
@@ -115,7 +115,7 @@ public void setName(String s) {
       box = new JCheckBox();
       box.setSelected(booleanValue());
       box.addItemListener(e -> setValue(box.isSelected()));
-      p.add(box);
+      p.add(box, "growx 0");
     }
 
```

Comment:
Might be better to revert this change as I have already made this change in master, but done it differently.

Reply:
reverted